### ISSUE TITLE
[WAC Virtual Machines and Switches] - Unable to hot-add memory to running VMs

### ISSUE THREAD

**Issue by ddcunningham** [MicrosoftDocs/Windows-Admin-Center-Ideas-and-Feedback#272]
**Gateway Version**: 1.3.2204.19002
**Extension Versions**
WAC version 2110.2
Virtual machines and switches 2.65.0
Failover cluster tools 2.81.0
Cluster Manager 2.120.0

**To Reproduce**
1. Navigate to Server Manager, Cluster Manager, or All connections.
2. Select a server running Hyper V 2016 and up (which is confirmed to support memory hot-add)
3. Navigate to Virtual Machines on the left sidebar, and select a given, powered on VM.
4. Navigate to that VM's settings view, then Memory.  Note that you cannot change any settings since the VM is powered on.

**Expected behavior**
The ability to hot-add memory to an online VM ( or perform a runtime memory resize ), should be an available feature within WAC if the underlying Hyper-V server supports it, from what I can tell online.  This seems to be a regressive bug.

**Screenshots & Additional context**
![chrome_2022-12-01_20-02-37](https://user-images.githubusercontent.com/6452523/205205659-e97d6c89-d359-4c66-ae76-631ace7ce941.png)

**Comment by RebeccaMbulaMs** [comment]
Hi @ddcunningham,

Is this issue still occurring on[ our latest build](https://github.com/MicrosoftDocs/Windows-Admin-Center-Ideas-and-Feedback/issues/287)?

Kind regards,
Rebecca

**Comment by RebeccaMbula** [comment]
No, no fix yet, but since you have confirmed it is still reoccurring, I will file a bug for more investigation. Thank you!

**Comment by NagareHoshi** [comment]
I checked this out and confirm the issue no longer exists.

**Comment by mattatmsft** [comment]
Thanks @NagareHoshi for the confirmation.  This issue should have been closed a while ago.  Doing so now.
